app/styles/_tables.less

```diff
@@ -210,3 +211,7 @@
     overflow-wrap: normal;
   }
 }
+
+.table-top-margin {
```

Comment:
Is there an advantage to adding this rule over using the existing spacer classes?